src/web/app/views/dialogs/ConfirmPrompt.tsx

```diff
@@ -77,6 +77,7 @@ function ConfirmPrompt({
         <div className="caption">{caption}</div>
         <pre className="message">{message}</pre>
         <input
+          id="power-caption"
```

Comment:
this is a general component, so id like `confirm-prompt-input` is better

Reply:
finished